src/internal/connector/onedrive/restore.go

```diff
@@ -210,20 +228,21 @@ func RestoreCollection(
 	restoreFolderElements := []string{restoreContainerName}
 	restoreFolderElements = append(restoreFolderElements, drivePath.Folders...)
 
+	ctx = clues.Add(
+		ctx,
+		"destination_elements", restoreFolderElements,
+		"drive_id", drivePath.DriveID)
+
 	trace.Log(ctx, "gc:oneDrive:restoreCollection", directory.String())
-	logger.Ctx(ctx).Infow(
-		"restoring to destination",
-		"origin", dc.FullPath().Folder(false),
-		"destination", restoreFolderElements)
+	logger.Ctx(ctx).Info("restoring onedrive collection")
```

Comment:
Should we be reducing the info in log messages?

Reply:
See line 228.  This data was already added to the context.